frontend/scripts/build.sh

```diff
@@ -1,12 +1,21 @@
 #!/usr/bin/env nix-shell
 #! nix-shell -j 4 -i bash -p stack git
-#! nix-shell -I nixpkgs=https://github.com/NixOS/nixpkgs/archive/464c79ea9f929d1237dbc2df878eedad91767a72.tar.gz
+#! nix-shell -I nixpkgs=https://github.com/NixOS/nixpkgs/archive/763e21e982370f67c126f92a1113ea949db3b6e0.tar.gz
+export NIX_PATH=nixpkgs=https://github.com/NixOS/nixpkgs/archive/763e21e982370f67c126f92a1113ea949db3b6e0.tar.gz
```

Comment:
It pins down the whole Nix package set to a commit: https://github.com/NixOS/nixpkgs/commit/763e21e982370f67c126f92a1113ea949db3b6e0